packages/grafana-ui/src/components/ContextMenu/ContextMenu.story.tsx

```diff
@@ -18,6 +19,14 @@ const meta: ComponentMeta<typeof ContextMenu> = {
     docs: {
       page: mdx,
     },
+    controls: {
+      exclude: ['renderMenuItems', 'renderHeader'],
```

Comment:
Let's exclude `onClose` as well from here

Reply:
Also the `focusOnOpen` is not helpful here because you can't open it. I'm wondering if we should add controls to the `WithState` story as well and that will then let us figure out what the `focusOnOpen` does.

Reply:
I excluded onClose, and also added controls for WithState to better display the effects of focusOnOpen, previously you could see it by refreshing the page, but that isn't very user friendly, now anyone viewing the stories can toggle it on and off and WithState will show the correct behavior. Let me know what you think!